public/assets/js/invite.js

```diff
@@ -1,12 +1,15 @@
 var invite = function() {
     var url = location.href;
     const localId = document.getElementById('js-local-id').innerText;
-
-    var inviteUrl = document.getElementById('js-invite-url');
+    if (localId == "No User"){
+        return;
+    }
```

Comment:
Fantastic! I didn't realize this method.

IMO(in my opinion), from the perspective of UX, before return, we should notify to user that the copy is failed.
The simplest case is like bellow code.
```case1
if (localId == "No User"){
        return;
    UIkit.notification({
        message: 'コピー失敗、IDが表示されてからもう一度招待ボタンを押して下さい',
        status: 'dan ger',
        pos: 'top-center',
        timeout: 2000
    });
}
```
Other case
1. first define `message` val (I think we can also initialize this val here, not only define)
2. just modify message `if localId==NoUser`
3. goto UIKit.notification() (it meens, skip process about copy the invite url to clipboard)

```case2
var message = 'コピー失敗、IDが表示されてからもう一度招待ボタンを押して下さい';
if (localId == "No User"){
    // goto UIKit.notification
}

```
IMO, Ideally we should modify the specification of invite button not to be able to click before available the SkyWay peer ID by adding 'disable' property, that is HTML5 power, to our invite button.

Could you try modifying your code to improve UX?

Reply:
Thanks for the review!!
I  totally agree about giving the notification when copy fails.
I'll fix it 👍